docs/reference.md

```diff
@@ -64,6 +64,65 @@ Ready message should be send by a plugins as soon as it renders their content an
 
 If ready message will not be sent within ~10 seconds, Ellipsis will render a timeout error and plugin content will be discarded.
 
+Request additional Story ANS
+
+Customers can obtain addition ANS data from Ellipsis via `postMessage`. Ellipsis/Composer will send back the data with the additional fields of:
+
+1. Headlines
+2. Subheadlines
+3. Taxonomy
+
+To acheive this:
+- Add an additional flag `isAnsRequired` set to `true` in your API where `postMessage()` is happening.
```

Comment:
same here, could u change `API` to integration to match the term used in this doc